shared/react-native/packageAndBuild.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# starts up the RN packager & requests both ios and android bundles
+
+set -e -u -o pipefail # Fail on error
+
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd $dir/..
+arg=${1:-}
+case $arg in 
+  "ios"|"android"|"both"|"")
```

Comment:
can just pass ios and android right? no need for 'both'

Reply:
also, i usually run this in a while true loop since it can crash out, maybe we should just add that as a flag too